src/main/java/lotto/service/LottoService.java

```diff
@@ -36,4 +51,15 @@ public MatchPointRank getMatchPointMap() {
     public List<Lotto> getLottoList() {
         return this.lottoList;
     }
+
+    private void invalidBonusNumber(List<Integer> integerList, int bonusNumber) {
+        if(bonusNumber < MIN_LOTTO_NUMBER || bonusNumber > MAX_LOTTO_NUMBER) {
```

Comment:
```suggestion
        if(bonusNumber < MIN_LOTTO_NUMBER || bonusNumber > MAX_LOTTO_NUMBER) {
```

`Number bonusNumber`이면 이미 위 검증이 되어있지 않을까요?

Reply:
<img width="324" alt="스크린샷 2023-11-25 오전 2 35 33" src="https://github.com/next-step/java-lotto/assets/12059592/c2805ddb-edc3-4a36-a6c9-3ee058895773">


(혹시 제 환경 세팅때문에 발생하는것일수 있어요) 실패 하는 테스트 확인 부탁드려요!

Reply:
도메인 객체 추가 후 제가 테스트를 돌려보지 않아 발생한 오류였어요!
수정하고 다시 PR 올립니다!

감사합니다!